src/Laravel/Html/FormFactory.php

```diff
@@ -102,7 +102,7 @@ public function createField(string $name, array $rules): FormField
             if (Str::startsWith($rule, ['min:', 'max:'])) {
                 [$attribute, $value] = explode(':', $rule, 2);
 
-                $attributes[$attribute] = $value + 0; //convert string to integer|float
+                $attributes[$attribute] = (int) $value; //convert string to integer|float
```

Comment:
This could be a float e.g., "step: 1.5" so casting to int could lead to issues. It just needs to be a numeric scalar e.g., float or int. Adding 0 was the simplest way of doing that without unnecessarily casting to float in all cases (e.g., integer-y strings would be cast to int)

Reply:
Added type checking against `float` or `int` before casting to `int`, otherwise, keep as is.

Reply:
I think in both cases it will start off as a string btw

Reply:
Do you think maybe we should just keep it as a string?